provider/terraform/examples.rb

```diff
@@ -107,6 +107,8 @@ class Examples < Api::Object
       # Whether to skip generating tests for this resource
       attr_reader :skip_test
 
+      attr_reader :skip_delete
```

Comment:
Is this skipping all deletes, or just skipping the test for deletion?

Reply:
This only skips the delete call in the generated test. The [custom delete](https://github.com/GoogleCloudPlatform/magic-modules/pull/3281/files#diff-6a7362a33a2f6b7a6896d550919a49a4R24) is what actually stops the resource from deleting.

Reply:
Could probably use a different name, or a comment maybe, or else you could genericize the custom delete?

Reply:
Oh, looks like it's already pretty generic.  you could maybe make this automatically add the custom delete and move the existing code around a bit to make it consistent?